### ISSUE TITLE
[QUESTION]:How to use default toolbar icon color?

### ISSUE THREAD

**Issue by littlestar2125** [mallowigi/material-theme-issues#548]
### Describe the bug

Hello,I'm new free user, i installed the plugin from Idea marketplace,now my toolbar run confirguration icons color is changed,I want to use default color,how to reset it?I jsut want to use editor theme,other settings i want to use idea default setting.
![image](https://github.com/user-attachments/assets/823467f7-3c18-48a0-b043-86de6260813c)

### To Reproduce

1.Installed the material theme ui

### Expected behavior

1.I want reset the default toolbar color,I can't find it.

### Material Theme Version

v7.0.0

### IDE

IntelliJ IDEA

### System Info

```shell
MacOs Idea
```

### Logs/Screenshots

```shell
No
```

**Comment by mallowigi** [comment]
I don't understand, from your screenshot it looks like the default behavior

**Comment by littlestar2125** [comment]
yes,but the top butoon color is not default color,i just want the top butoon is default behavior,it sign'd by blue line.
![企业微信截图_a05f2e52-ee1e-4083-9968-33bcf801438d](https://github.com/user-attachments/assets/c3e616d1-08bc-463b-8e6d-803cb80ade04)

**Comment by littlestar2125** [comment]
> I don't understand, from your screenshot it looks like the default behavior

yes,but the top butoon color is not default color,i just want the top butoon is default behavior,it sign'd by blue line.

![image](https://github.com/user-attachments/assets/e763afb2-eaf2-4769-8650-3926a5fe9487)

**Comment by mallowigi** [comment]
No idea, are you sure this is a plugin issue?

**Comment by Dimiikou** [comment]
I guess he is referring to the different colored Icons, the ones in the Main Window are light-blue and the ones in the Dropdown are colored Green as the IDE Default i guess

**Comment by Dimiikou** [comment]
He want's to know how to set the color of the light-blue Icon to the IDEA default Theme settings, as done for the Run and Debug icons inside the Dropdown menu. His goal is to only apply the theme to his editor window itself without the toobar etc.

**Comment by mallowigi** [comment]
Sorry, this will not be configurable. The buttons have the accent color like other components of the UI. The green icons you see on the popup are not accented, and this is a decision made by the JetBrains team. But I won't make a setting just for that single use.
